### ISSUE TITLE
[Bug] shadow window

### ISSUE THREAD

**Issue by AlessandroVeller** [JasonStein/Notepads#662]
**Describe the bug**
Every time I open the app a message pops up saying "this is a shadow window of notepads. Session snapshot and settings are disabled." Am I doing something wrong or is this a bug?

**Comment by KristofKekesi** [comment]
I have this bug too.

**Comment by JasonStein** [comment]
This will be fixed in v1.4.0.0

**Comment by PkW4v3** [comment]
I have version 1.4.2.0 but i still have this bug. Can you help me? @JasonStein

**Comment by kushaagr** [comment]
Issue persists in version 1.4.2.0
![2020 12 24](https://user-images.githubusercontent.com/68564934/103039129-15ea0000-4596-11eb-895a-9593e642695e.png)

**Comment by soumyamahunt** [comment]
@PkW4v3 @PkW4v3 can you provide a video recording of launching notepads with the task manager window included. Also expand the "Notepads" node in task manager and show the changes for that node in the video.

**Comment by 706B** [comment]
I am also facing this issue. I already have notepads on my desktop which works fine. I just installed it on my laptop today and I am getting this message every time I open the app. Just after the install, it worked fine but after that (from the second time I opened the app), I have been getting this message. the setting is also not accessible now in the app.

one noticeable difference between my laptop and desktop is that my desktop is on window10 2004 version and my laptop in on 20H2 (latest ver.)

![image](https://user-images.githubusercontent.com/14940852/103410704-68ae5380-4b64-11eb-87c0-3033ed9194a6.png)
